llpc/test/shaderdb/gfx9/PipelineVsFs_test_cache_alpha_to_coverage_disabled.pipe

```diff
@@ -0,0 +1,276 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -auto-layout-desc -spvgen-dir=%spvgendir% -v %gfxip --use-relocatable-shader-elf -shader-cache-mode=2 -shader-cache-filename=cache.bin -build-shader-cache test_cache_vert.spv test_cache_frag.spv
+; RUN: amdllpc -auto-layout-desc-spvgen-dir=%spvgendir% --use-relocatable-shader-elf -shader-cache-filename=cache.bin -shader-cache-mode=4 -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: Building pipeline with relocatable shader elf.
+; SHADERTEST: Cache hit for shader stage 0
+; SHADERTEST: Cache hit for shader stage 4
+; SHADERTEST: AMDLLPC SUCCESS
+; END_SHADERTEST
```

Comment:
Does anybody have any idea on how I could do this test?  I want to run `amdllpc` with the shader files individually first to create a cache.  Then run the .pipe file through `amdllpc` to make sure there is a cache hit in the relocatable shader cache.  It currently fails because it cannot find the shader file in the first run of `amdllpc`.

Reply:
One option would be to create a custom script that runs things in the order you described. You can then add it as a custom target in the lit configuration.

Reply:
Minor correction: lit calls these substitutions tools. You can find examples in: `llvm-project/llvm/test/lit.cfg.py`.

Reply:
llvm-lit is capable of running a sequence of commands as part of one test. Unfortunately I don't know the details, but I have run across tests in LLVM (around MC etc.) that do this, so I suggest digging into that.